src/pwr.ps1

```diff
@@ -1192,10 +1318,24 @@ switch ($Command) {
 			}
 		}
 	}
+	'prune' {
+		if ($Packages.Count -gt 0) {
+			Write-PwrFatal 'prune not compatible with -Packages'
+		}
+		Lock-PwrLock
+		try {
+			$LoadCache = Get-PwrLoadCache
+			Prune-PwrPackages $LoadCache
+			Save-PwrLoadCache $LoadCache
+		} finally {
+			Unlock-PwrLock
+		}
+	}
 	{$_ -in 'rm', 'remove'} {
```

Comment:
Any side effects of not updating the load cache on remove?

Reply:
The cache gets automatically cleaned up when a `prune` is run (lines 1086-1100). Having something extra in there doesn't hurt, since the packages in the cache are never used to look up anything. It's just an extraneous item until it gets cleaned up.